src/infrasys/time_series_models.py

```diff
@@ -54,6 +54,7 @@ class TimeSeriesStorageType(str, Enum):
 class TimeSeriesData(InfraSysBaseModelWithIdentifers, abc.ABC):
     """Base class for all time series models"""
 
+    data: Any  # TODO update to generic?
```

Comment:
Isn't this defined in the child classes?

Reply:
We can let the child class set the name and type of their data attribute. My understanding is that no current code accesses an attribute called `data` if the type is generically `TimeSeriesData`. It should only be accessing that attribute if it knows that it is of type `SingleTimeSeries`. Is that right?